runtime/src/bank/partitioned_epoch_rewards/calculation.rs

```diff
@@ -512,6 +513,72 @@ impl Bank {
 
         (points > 0).then_some(PointValue { rewards, points })
     }
+
+    /// If rewards are active, recalculates partitioned stake rewards and stores
+    /// a new Bank::epoch_reward_status. This method assumes that vote rewards
+    /// have already been calculated and delivered, and *only* recalculates
+    /// stake rewards
```

Comment:
Cool, just to be clear I think we can commit to keeping calculation in the first block for now.. SIMD-0118 doesn't mention that the number of calculation blocks is configurable at all. So I'm strongly in favor of ripping out anything that makes it configurable and lean into assuming that it's a single block for now and I'm not concerned about any brittleness that comes from that assumption. If we ever decide to extend it over multiple blocks, it will require a new SIMD and careful implementation beyond just updating a constant.

Reply:
Follow-up here: https://github.com/anza-xyz/agave/pull/1251